geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/ingest/AbstractIngest.scala

```diff
@@ -193,24 +193,45 @@ object AbstractIngest {
                     emptyBar: String,
                     replacement: Char,
                     indicator: Char)(progress: Float, start: Long, pass: Long, fail: Long, done: Boolean): Unit = {
-    val numFilled = (emptyBar.length * progress).toInt
-    val bar = if (numFilled < 1) {
-      emptyBar
-    } else if (numFilled >= emptyBar.length) {
-      buildString(replacement, numFilled)
+    val percent = f"${(progress * 100).toInt}%3d"
+    val infoStr = s" $percent% complete $pass ingested $fail failed in ${getTime(start)}"
+
+    // Figure out if and how much the progress bar should be scaled to accommodate smaller terminals
+    val scaleFactor: Float = try {
+      val tWidth: Float = jline.Terminal.getTerminal.getTerminalWidth.toFloat
```

Comment:
does jline have any better options than using `\r` to replace the current line? Maybe we should have 2 methods - one the enhanced jline version and one the original basic version in case there's no jline.